sway/ipc-server.c

```diff
@@ -154,6 +157,7 @@ int ipc_handle_connection(int fd, uint32_t mask, void *data) {
 		return 0;
 	}
 
+#ifdef __linux__
```

Comment:
Don't comment this out, just have the FreeBSD version of get_client_pid return -1.